jsx-runtime not found error in downstream vitest project

- related? https://github.com/facebook/react/issues/20235

The issue is found in:

- https://github.com/hi-ogawa/ytsub-v3/pull/171

Currently we use `"jsx": "react-jsx"` for `@hiogawa/utils-react` package.
React version (ytsub-v3 uses react 17) might be relevant.

---

## logs

https://github.com/hi-ogawa/ytsub-v3/actions/runs/4401061028/jobs/7706928767#step:7:66

```
 FAIL  app/routes/__tests__/videos-id.test.ts [ app/routes/__tests__/videos-id.test.ts ]
Error: Cannot find module '/home/runner/work/ytsub-v3/ytsub-v3/node_modules/.pnpm/@hiogawa+utils-react@1.1.0_react@17.0.2/node_modules/react/jsx-runtime' imported from /home/runner/work/ytsub-v3/ytsub-v3/node_modules/.pnpm/@hiogawa+utils-react@1.1.0_react@17.0.2/node_modules/@hiogawa/utils-react/dist/index.js
Did you mean to import react@17.0.2/node_modules/react/jsx-runtime.js?
```

--- a/.changeset/nice-trees-cough.md
+++ b/.changeset/nice-trees-cough.md
@@ -1,0 +1,5 @@
+---
+"@hiogawa/utils-react": patch
+---
+
+chore(utils-react): package without jsx-runtime

--- a/packages/utils-react/src/debug.tsx
+++ b/packages/utils-react/src/debug.tsx
@@ -1,3 +1,5 @@
+import React from "react";
+
 export function Debug(
   props: { debug: unknown } & JSX.IntrinsicElements["details"]
 ) {

--- a/packages/utils-react/tsconfig.json
+++ b/packages/utils-react/tsconfig.json
@@ -7,6 +7,7 @@
     "module": "ESNext",
     "target": "ESNext",
     "lib": ["ESNext", "DOM"],
-    "jsx": "react-jsx"
+    // react-jsx would require tricky workaround in old react https://github.com/hi-ogawa/js-utils/issues/19
+    "jsx": "react"
   }
 }

